src/utils/userauth.tsx

```diff
@@ -0,0 +1,78 @@
+import * as auth0 from 'auth0-js'
+import { navigateTo } from 'gatsby-link'
+
+export default class UserAuth {
+  auth0 = new auth0.WebAuth({
+    audience: 'https://product-mindset.auth0.com/api/v2/',
+    clientID: 'dQyBO6WzKfWj9PEEq8LZNIjeJvpRmXau',
+    domain: 'product-mindset.auth0.com',
+    redirectUri: 'http://localhost:8000/callback',
```

Comment:
This will need the actual redirect uri, not localhost for production.

Can probably use https://www.netlify.com/docs/netlify-toml-reference/ and https://www.netlify.com/docs/continuous-deployment/#build-environment-variables